nbgrader/docs/source/user_guide/advanced.rst

```diff
@@ -11,71 +11,6 @@ Running nbgrader with JupyterHub
 
 Please see :doc:`/configuration/jupyterhub_config`.
 
-.. _assignment-list-installation:
```

Comment:
This is already covered in the main installation page. Is that sufficient or do you want to also keep it in this location? The benefit of the main location is that it covers global installation of both the server extension and nbextensions, rather than just the assignment-list stuff. More than willing to put a note back about this.